brion/spikeReport.cpp

```diff
@@ -60,13 +44,31 @@ class SpikeReport
         plugin.reset( SpikePluginFactory::getInstance().create( initData ));
     }
 
-    boost::scoped_ptr< SpikeReportPlugin > plugin;
+public:
+    std::unique_ptr< SpikeReportPlugin > plugin;
+    lunchbox::ThreadPool threadPool {1};
```

Comment:
It may be useful to have a thread pool in Lunchbox, but since you can have at most one asynchronous operation going on at the same time, what's the benefit of using a thread pool here instead of std::async?